### PR TITLE
Default Epic back to control message

### PR DESCRIPTION
**Pull request by philwills**
## What does this change?

Removes the epic contributions/subscriptions ask everywhere except the US, defaults to the control message, which was once again victorious and adds a supporter only call-to-action to the testing mix.

## What is the value of this and can you measure success?

We aim to have a clear idea of whether asking for contributions, sign-ups to membership, or a mix of the two is the most valuable way of asking people to support the Guardian in the US.

## Does this affect other platforms - Amp, Apps, etc?
No

## Request for comment

cc @jranks123

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js [hunk 1]
```diff
@@ -33,32 +33,36 @@ define([
 
     return function () {
 
-        this.id = 'ContributionsEpicUsaCtaFakeNews';
+        this.id = 'ContributionsEpicUsaCta';
```
Comment (jranks123) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js]: We have already named a test with this name (https://github.com/guardian/frontend/pull/15031/), so we should give it a distinctive name so it's easy to analyse the results

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js [hunk 2]
```diff
@@ -244,14 +205,14 @@ define([
...
                 test: function () {
-                    var ctaType = cta.justContribute;
-                    var message = messages.fake;
+                    var ctaType = cta.justSupporter;
+                    var message = messages.control;
                     var component = $.create(template(contributionsEpicEqualButtons, {
-                        linkUrl1: ctaType.url1 + message.intcmp + '_contribute',
-                        linkUrl2: ctaType.url2 + message.intcmp + '_contribute',
+                        linkUrl1: ctaType.url1 + message.intcmp + '_supporter',
+                        linkUrl2: ctaType.url2 + message.intcmp + '_supporter',
```
Comment (jranks123) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js]: message.intcmp will always be "_control" in this test, so isn't needed for the campaign code for any of the variants

Comment (jranks123) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js]: Also, the ctaType.url1 and ctaType.url2 for just supporter and just contribute can lose the "_backup" now, as they are no longer being used as backups

**Review by jranks123 — COMMENTED** [review]
Looks good to me with comments, but get a dotcom pair of eyes as well 👍

**Comment by philwills** [comment]
Thanks, @jranks123, I think I've addressed all those points, do you mind taking a second look?

**Review by katebee — APPROVED** [review]
Looks good to me 👀 👍

**Pull request merged**